langlib/lang.string/src/main/ballerina/string.bal

```diff
@@ -275,6 +376,11 @@ public isolated function fromCodePointInt(int codePoint) returns Char|error = @j
 # Adds sufficient `padChar` characters at the start of `str` to make its length be `len`.
 # If the length of `str` is >= `len`, returns `str`.
 #
+# ```ballerina
+# // Providing "0" as the custom padding char
+# string distanceAsFixedLengthString = "100Km".padStart(10, "0");
```

Comment:
We need another example with `padChar` with default value

Reply:
I feel it is not required because it is already covered in parameter description.